platform/cert/cert_manager_test.go

```diff
@@ -385,6 +385,35 @@ var _ = Describe("Certificate Management", func() {
 			})
 		})
 
+		Context("PhotonOS", func() {
+			BeforeEach(func() {
+				fakeFs = fakesys.NewFakeFileSystem()
+				fakeCmdRunner = fakesys.NewFakeCmdRunner()
+				fakeCmdRunner.AddCmdResult("/usr/bin/rehash_ca_certificates.sh", fakesys.FakeCmdResult{
+					Stdout:     "",
+					Stderr:     "",
+					ExitStatus: 0,
+					Sticky:     true,
+				})
+				certManager = cert.NewPhotonOSCertManager(fakeFs, fakeCmdRunner, 0, log)
+			})
+
+			SharedLinuxCertManagerExamples("/etc/ssl/certs", "/usr/bin/rehash_ca_certificates.sh")
+
+			It("executes update cert command", func() {
```

Comment:
This test doesn't pass when we ran it locally. Can you fix this?

Reply:
Fixed.